src/pages/PeopleTable.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { Person } from '../types';
+import { PersonInfo } from './PersonInfo';
+
+interface Props {
+  people: Person[]
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const normalizedPeople = people.map(person => {
+    return {
+      ...person,
+      father: people.find(father => father.name === person.fatherName),
+      mother: people.find(mother => mother.name === person.motherName),
+    };
```

Comment:
great desicion! 🔥 I like it

also add check `|| null`
so if you dont find any parent its better to have `null` value instead of `undefined`

and you can move this function to `utils` folder